.travis.yml

```diff
@@ -6,10 +22,56 @@ matrix:
           dist: trusty
           sudo: required
           env: TRAVIS_BUILD_TYPE=Debug
+          env: CMAKE_OPTIONS="-DCMAKE_C_COMPILER=/usr/lib/ccache/$CC -DCMAKE_CXX_COMPILER=/usr/lib/ccache/$CXX"
```

Comment:
Yeah - I am not sure on that either.   I've removed it since.   We can play around with it more in the future.